src/kagglehub/cache.py

```diff
@@ -17,8 +19,9 @@ def load_from_cache(handle: Union[ModelHandle], path: Optional[str] = None) -> O
     Returns:
         A string representing the path to the requested resource in the cache or None on cache miss.
     """
+    marker_path = _get_completion_marker_filepath(handle, path)
     full_path = get_cached_path(handle, path)
-    return full_path if os.path.exists(full_path) else None
+    return full_path if os.path.exists(marker_path) and os.path.exists(full_path) else None
```

Comment:
Should there be a way to force cache miss in case you want to get the latest file? (only necessary if we allow users to download an instance instead of a instanceVersion which is immutable)

Reply:
I can add a --force flag later.

FYI: If the user don't specify a version, we will first do a look up to get the latest version and then call this logic. This means, newer model will be downloaded automatically (not yet implemented)